src/tasks/tasks.validators.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNotEmpty, IsNumberString } from 'class-validator';
+
+export type Task = {
+    id: number;
+    status: string;
+    taskName: string;
+    startDate: Date;
+    finishDate: Date;
+  };
+
+
+export class ValidateId {
+  @IsNotEmpty()
+  @IsNumberString()
+  id: number;
+}
+
+export class RequestIsNotEmpty {
+    @IsNotEmpty() id: number;
+    @IsNotEmpty() status: string;
```

Comment:
Maybe we need to add some more "validation" to the fileds? Not just **IsNotEmpty** ?